### PR TITLE
Add Wikia

### PR DESCRIPTION
**Pull request by invalidCards**
After one of our admins got hacked, we're mad at Wikia for not having 2FA

### CODE REVIEW

**Comment by RichJeanes** [comment]
So are these accounts for any site that runs on the Wikia platform?

**Comment by invalidCards** [comment]
Yes, although 99% of those sites run on a wikia.com subdomain (eg. runescape.wikia.com).

File: _data/other.yml [hunk 1]
```diff
@@ -285,6 +285,12 @@ websites:
       software: Yes
       doc: http://doxfer.webmin.com/Webmin/Enhanced_Authentication
 
+    - name: Wikia
+      url: https://www.wikia.com/
+      twitter: Wikia
+      img: wikia.png
+      tfa: no
```
Comment (Carlgo11) [inline _data/other.yml]: Please use a capital `N` in `tfa: no`

File: _data/other.yml [hunk 2]
```diff
@@ -285,6 +285,12 @@ websites:
       software: Yes
       doc: http://doxfer.webmin.com/Webmin/Enhanced_Authentication
 
+    - name: Wikia
+      url: https://www.wikia.com/
+      twitter: Wikia
```
Comment (Carlgo11) [inline _data/other.yml]: Looks like their twitter handle is all lowercase. So `wikia`.

File: _data/other.yml [hunk 3]
```diff
@@ -285,6 +285,12 @@ websites:
       software: Yes
       doc: http://doxfer.webmin.com/Webmin/Enhanced_Authentication
 
+    - name: Wikia
+      url: https://www.wikia.com/
```
Comment (Carlgo11) [inline _data/other.yml:289]: That URL redirects to `http://www.wikia.com/fandom`. Is that just me or is that really their URL?

Comment (invalidCards) [inline _data/other.yml:289]: I wasn't sure which to put here - `www.wikia.com/fandom` is their kinda new project to relate more to the people making and maintaining the wikis whereas `http://community.wikia.com` is their "starting point" into making wikis, so to speak.

**Pull request merged**